Library/Formula/pidcat.rb

```diff
@@ -3,8 +3,8 @@
 class Pidcat < Formula
   homepage "https://github.com/JakeWharton/pidcat"
   head "https://github.com/JakeWharton/pidcat.git"
-  url "https://github.com/JakeWharton/pidcat/archive/1.4.1.tar.gz"
-  sha1 "89f806ae1fa3375ce188851c8c95fc1097467b82"
+  url "https://github.com/JakeWharton/pidcat/archive/2.0.0.tar.gz"
+  sha1 "7c33f7653894eccc9296a16f71fcf61cdee56cde"
```

Comment:
If you wanted to bump this to `sha256`, :tada:, but it's not mandatory for existing formulae.